package.json

```diff
@@ -9,8 +9,9 @@
   "scripts": {
     "fix": "eslint --fix .",
     "lint": "eslint .",
+    "spelling": "cspell examples/**/**/**/**/*.html",
```

Comment:
yeah, I've since figure'd out that the globs do work properly as long as there is a double quote around them.
EX:
Doesn't work `cspell examples/**/*.html`
Does work `cspell "examples/**/*.html"`